### PR TITLE
Fix block breaking

### PR DESCRIPTION
**Pull request by Thodor12**
- Removed mineable tag from every materially textured block
- Removed "requiresToolForDrops" from every materially textured block
- Added proper mineable tag to "extra" blocks
- Renamed "clay" extra blocks to "bricks", to better refer the final output material
- Fixed (fancy) door block drops (double dropping doors / creative drops)
- Fixed slab block drops (dropping single items with double slab type)
- Removed possibility of mixing different slab types

### CODE REVIEW

File: src/datagen/generated/domum_ornamentum/data/minecraft/tags/blocks/doors.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
   "values": [
-    "domum_ornamentum:vanilla_doors_compat"
+    "domum_ornamentum:fancy_door"
```
Comment (Raycoms) [inline src/datagen/generated/domum_ornamentum/data/minecraft/tags/blocks/doors.json:4]: make sure this doesnt break stuff

File: src/datagen/generated/domum_ornamentum/data/minecraft/tags/blocks/mineable/shovel.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+  "values": [
+    "domum_ornamentum:black_brick_extra",
+    "domum_ornamentum:blue_brick_extra",
+    "domum_ornamentum:brown_brick_extra",
+    "domum_ornamentum:brick_extra",
+    "domum_ornamentum:cyan_brick_extra",
+    "domum_ornamentum:gray_brick_extra",
+    "domum_ornamentum:green_brick_extra",
+    "domum_ornamentum:light_blue_brick_extra",
```
Comment (Raycoms) [inline src/datagen/generated/domum_ornamentum/data/minecraft/tags/blocks/mineable/shovel.json]: shovel?

File: src/datagen/generated/domum_ornamentum/data/minecraft/tags/blocks/wooden_doors.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
   "values": [
-    "domum_ornamentum:vanilla_doors_compat"
+    "domum_ornamentum:fancy_door"
```
Comment (Raycoms) [inline src/datagen/generated/domum_ornamentum/data/minecraft/tags/blocks/wooden_doors.json:4]: same

File: src/datagen/generated/domum_ornamentum/data/minecraft/tags/blocks/mineable/shovel.json (hunk 2 above)
Comment (Thodor12) [reply src/datagen/generated/domum_ornamentum/data/minecraft/tags/blocks/mineable/shovel.json]: Oh wait that should be pickaxe, I'm going to rename the source from "clay" to "brick'

File: src/datagen/generated/domum_ornamentum/assets/domum_ornamentum/models/block/extra/brick/orange_brick_extra.json
Comment (Raycoms) [inline src/datagen/generated/domum_ornamentum/assets/domum_ornamentum/models/block/extra/brick/orange_brick_extra.json:1]: Did you test back forwards compat of block?

**Review by marchermans — APPROVED** [review]

**Pull request merged**